recipes/zfit/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "zfit" %}
+{% set version = "0.3.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 26e76eb100c95ed52241f3b552d7dd16f59091a83f5e01b263f6fa9f12b30cfe
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools_scm
+    - setuptools_scm_git_archive
+  run:
+    - python >=3.6
+    - tensorflow-base >=1.14.0
+    - tensorflow-probability >=0.6.0
+    - scipy >=1.2
+    - uproot
+    - pandas
+    - numpy
+    - iminuit
+    - typing
+    - colorlog
+    - texttable
+    # Workaround for https://github.com/conda-forge/tensorflow-probability-feedstock/pull/11
+    - decorator
+    - cloudpickle >=0.6.1
+
+test:
+  imports:
+    - zfit
+    - zfit.core
+    - zfit.minimizers
+    - zfit.models
+    - zfit.util
+    - zfit.ztf
+
+about:
+  home: https://github.com/zfit/zfit
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: Physics extension to zfit
+  doc_url: https://zfit.readthedocs.io/
+  dev_url: https://github.com/zfit/zfit
+
+extra:
+  recipe-maintainers:
+    - chrisburr
+    - mayou36
```

Comment:
@mayou36 Can you confirm here that you're happy to be listed as a maintainer?

Reply:
I confirm ;)